data/org.freedesktop.portal.SpeechSynthesis.xml

```diff
@@ -0,0 +1,169 @@
+          object path element. See the :ref:`org.freedesktop.portal.Request` documentation for
+          more information about the @handle.
+
+        The following results get returned via the :ref:`org.freedesktop.portal.Request::Response` signal:
+
+        * ``providers`` (``a(ss)``)
+
+           An array of providers. Each provider in the array is structure with the following members:
+             * A well known name
+             * A human readable name
+    -->
+    <method name="GetProviders">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="parent_window" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        GetVoices:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+
+        Get available synthesis voices
+
+        Supported keys in the @options vardict include:
+
+        * ``handle_token`` (``s``)
+
+          A string that will be used as the last element of the @handle. Must be a valid
+          object path element. See the :ref:`org.freedesktop.portal.Request` documentation for
+          more information about the @handle.
+
+        The following results get returned via the :ref:`org.freedesktop.portal.Request::Response` signal:
+
+        * ``voices`` (``a(ssstas)``)
+
+          An array of voices. Each voice in the array is structure with the following members:
+            * A human readable name
+            * A unique identifier
+            * Synthesis output format
+            * A voice features bit field
+            * A list of languages the voice support represented as BCP 47 tags
+    -->
+    <method name="GetVoices">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="parent_window" direction="in"/>
+      <arg type="s" name="provider_id" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        Synthesize:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+        @pipe_fd: File descriptor of pipe to write to.
+        @text: The text to be spoken.
+        @voice_id: The voice identifier for the voice that should be spoken.
+        @pitch: The voice pitch in which the text should be spoken.
+        @rate: The rate in which the text should be spoken.
+        @is_ssml: True if the text should be interpretted as an SSML snippet.
```

Comment:
what is an ssml snippet?

Reply:
https://www.w3.org/TR/speech-synthesis/

Reply:
would be nice to document it there or at least link to this.